Remotion/Data/DomainObjects.ObjectBinding.IntegrationTests/DatabaseConfiguration.cs

```diff
@@ -24,48 +24,21 @@ namespace Remotion.Data.DomainObjects.ObjectBinding.IntegrationTests
 {
```

Comment:
This will also need to be done in relinq SqlBackend. do you want to create the ticket for this now or wait until later to remember and create the ticket?